tensorflow_serving/model_servers/server_core.h

```diff
@@ -117,6 +117,9 @@ class ServerCore : public Manager {
     // Time interval between file-system polls, in seconds.
     int32 file_system_poll_wait_seconds = 30;
 
+    // Timeout for waiting on a servale to reach state.
```

Comment:
Let's make the comment:
// Timeout for waiting for the models to load when the server starts or changes config.
(in this layer of our system we use the term "models" not "servables"; and the timeout is only on start and config change)
... and name the variable:
startup_and_config_reload_model_load_timeout (or something more concise if you can think of something :)